src/components/Login/Login.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styles from './login.module.scss';
+import LoginWelcomeBlock from './LoginWelcomeBlock/LoginWeclomeBlock';
+import LoginForm from './LoginForm/LoginForm';
+
+const Login = () => (
+  <div className={styles.loginContainer}>
+    <img className={styles.logo} src="/images/icon.png" alt="logo" />
+    <LoginWelcomeBlock />
+    <LoginForm />
+  </div>
+);
+
+export default Login;
```

Comment:
Why not to this call file index.js so we can import it as ... './components/Login' but not './components/Login/Login.js'